2018/q1/product-designer.md

```diff
@@ -1,42 +1,44 @@
-better software? We have insights into the quality, security,
-performance, and popularity of every package: how do we surface this
-to developers looking for packages? We know what the software
-development team at a company is building, how do we help management
-get a bigger picture?
+We believe that high-performing teams include people from different backgrounds and experiences who can challenge each other's assumptions with fresh perspectives. To that end, we actively seek a diverse pool of applicants, including those from historically marginalized groups — women, people with disabilities, people of color, formerly incarcerated people, people who are lesbian, gay, bisexual, transgender, and/or gender nonconforming, first and second generation immigrants, and people from low-income families.
 
-To solve these problems and give our users the best possible
-experience, npm is hiring a Senior Product Designer. Your job will be
-to take a holistic view of the entire npm experience, from the website
-to the command line client, and create cohesive and enjoyable
-experiences that let developers get real work done while giving
-managers insight into how software is being written. npm's ambition is
-to reduce friction for everyone in software development, and that will
-be your guiding purpose.
+*our code of conduct*
+npm exists to facilitate sharing code, by making it easy for JavaScript module developers to publish and distribute packages.
+npm is a piece of technology, but more importantly, it is a community.
+We believe that our mission is best served in an environment that is friendly, safe, and accepting; free from intimidation or harassment. We do not tolerate abusive behavior.
+See our unabridged code of conduct [here](https://www.npmjs.com/policies/conduct).
 
-You'll be tasked with getting inside our users' heads, applying
-empathy tactically to the puzzle of delivering a product vision within
-the constraints of what's possible in reality. While you might not be
-a software developer yourself, you are familiar with their experience
-and the challenges they face.
-
-You'll report directly to our head of product (who is also our CEO),
-and work closely with our engineering team as well as other major
-stakeholders like marketing, sales, and support. Your output will
-include workflows, information architecture, written product specs and
+*about this role:*
+You'll work closely with our head of product (who is also our CEO),
+and with our engineering team as well as other major
+stakeholders like marketing, sales, and support to improve current products
+and envision new ones to add value for our users. You'll help define and drive forward npm's product strategy as well as produce workflows, information architecture, written product specs and
 wireframes as appropriate.
 
 This position will be based in our offices in Oakland, CA, and we are
 not looking for remote candidates for this position.
+
```

Comment:
I dislike this bullet list. The "love this" list is not as expressive as the paragraph it replaces, and the "helpful" list is the kind of "degrees preferred" language that makes people not apply -- they will believe they need to have experience at all of these things to bother applying, and therefore less self-confident applicants will be dissuaded. This is a subtle but real barrier to more diverse hiring.

Reply:
The bulleted lists here are something I actually learned to use as a good practice for writing inclusive job descriptions - the love list shows that we are interested in characteristics over than experience, and the experience list helps people anchor themselves as a viable candidate while specifically not being a required quals list. That said, we can remove if needed but it is also helpful for many folks to have some visual breakup from paragraphs :)